src/components/published-modal.cjsx

```diff
@@ -0,0 +1,46 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+_ = require 'underscore'
+
+{ExerciseStore} = require '../stores/exercise'
+
+
+PublishedModal = React.createClass
+  getInitialState: -> {}
+
+  componentWillMount: ->
+    ExerciseStore.on('published', @show)
+
+  componentWillUnmount: ->
+    ExerciseStore.off('published', @show)
+
+  show: ->
+    @setState(isShowing: true)
+
+  onHide: ->
+    @setState(isShowing: false)
```

Comment:
Does this still work if the user publishes multiple exercises? (ie does the state get reset somehow)